app/services/service_listeners/publishing_api_pusher.rb

```diff
@@ -7,6 +7,10 @@ def initialize(edition)
     end
 
     def push(event:, options: {})
+      if edition.locked?
+        raise RuntimeError, "Cannot send a locked document to the Publishing API"
```

Comment:
RuntimeError is actually superfluous here as a raise will default to RuntimeError.

I do wonder if this should/could be a specific exception.